protocol/src/main/kotlin/Mapper.kt

```diff
@@ -1,25 +1,18 @@
 import com.fasterxml.jackson.databind.ObjectMapper
-import com.fasterxml.jackson.databind.ObjectReader
 import com.fasterxml.jackson.databind.ObjectWriter
 import com.fasterxml.jackson.databind.SerializationFeature
 import com.fasterxml.jackson.module.kotlin.registerKotlinModule
 
 const val dateTimeFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSXXX"
 
 object Mapper {
-    // TODO make private and add method for calling readTree
-    val objectMapper: ObjectMapper = ObjectMapper()
-        .registerKotlinModule()
-        .findAndRegisterModules()
-        .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
-
-    fun writer(): ObjectWriter {
-        return objectMapper.writer()
-    }
-
-    fun <T> reader(type: Class<T>): ObjectReader {
-        return objectMapper.readerFor(type)
-    }
-
+  // TODO make private and add method for calling readTree
+  val objectMapper: ObjectMapper = ObjectMapper()
```

Comment:
Curious why you went with jackson for json serialization vs [kotlinx serialization](https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/serialization-guide.md)?

Reply:
I think it's just what we've been using. Is there an advantage to using kotlinx instead?

Reply:
The two advantages I know of:
* Speed
* Verbosity
I don't know how well it's supported in different versions of kotlin though, or different platforms.

Reply:
Relevant comment https://github.com/TBD54566975/tbdex-kt/pull/31#issuecomment-1747426398

Reply:
@andresuribe87 , this was a grandfathered decision from @bradleydwyer and @tomdaffurn. I personally wanted to be able to use kotlinx, but also heard from @alecthomas that it wasn't ready for prime time. Kotlinx has a MUCH nicer interface.

Reply:
Yeah, I tried and failed to use kotlinx serialisation. I really liked the API too, but IIRC it was just generally quite restrictive and fiddly to use.